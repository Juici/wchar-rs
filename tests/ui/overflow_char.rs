use wchar::wch;

const OVERFLOW_U16: u16 = wch!(u16, 'ðŸ’–');
const OVERFLOW_I16: i16 = wch!(i16, 'ðŸ’–');

fn main() {}
