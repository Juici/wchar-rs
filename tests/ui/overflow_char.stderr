error: character does not fit within one u16 wide character
 --> $DIR/overflow_char.rs:3:37
  |
3 | const OVERFLOW_U16: u16 = wch!(u16, 'ðŸ’–');
  |                                     ^^^^

error: character does not fit within one i16 wide character
 --> $DIR/overflow_char.rs:4:37
  |
4 | const OVERFLOW_I16: i16 = wch!(i16, 'ðŸ’–');
  |                                     ^^^^
